web/static/js/configs/happy_sad_confused_stage_configs.js

```diff
@@ -112,6 +113,13 @@ export default {
       copy: "Proceed to Action Items",
       iconClass: "arrow right",
       confirmationMessage: "Are you sure you would like to proceed to the action items stage?",
+      stateDependentTooltip: reduxState => {
+        const { votes, presences } = reduxState
+        if (votes.length >= VOTE_LIMIT * presences.length) {
```

Comment:
@pwjablonski This isn't a blocker to merge, but we can make this inline logic more immediately grokkable to future maintainers by extracting `VOTE_LIMIT * presences.length` out into a constant that expresses the concept. Something like, `TOTAL_VOTE_LIMIT_FOR_RETRO`.

Again, not a blocker. I'll add this after the fact.